### PR TITLE
fix(odin): updates queries from upstream

### PR DESCRIPTION
**Pull request by laytan**

### CODE REVIEW

File: queries/odin/indents.scm [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+; hello(
+((identifier) . (ERROR "(" @indent.begin))
+
 [
-  "}"
   ")"
   "]"
 ] @indent.branch @indent.end
 
+; Have to do all closing brackets separately because the one for switch statements shouldn't end.
```
Comment (lucario387) [inline queries/odin/indents.scm:22]: Noice.

**Comment by lucario387** [comment]
Please run `./scripts/format-queries.lua queries/odin` to re-format the files

**Comment by clason** [comment]
And please rebase on `master`!

**Comment by laytan** [comment]
Done both!

File: queries/odin/highlights.scm [hunk 2]
```diff
@@ -151,6 +152,10 @@
   (identifier) @type
   "::")
 
+(bit_field_declaration
+  (identifier) @type
+  "::")
+
 (const_declaration
   (identifier) @type
```
Comment (lucario387) [inline queries/odin/highlights.scm:160]: We're getting a little bit too much of this pattern that I want to ask, can this be possible
```suggestion
((identifier) @type
  .
  "::")
```

Comment (laytan) [reply queries/odin/highlights.scm:160]: I think that would match constants `foo :: 1` and highlight it as a type too

Comment (lucario387) [reply queries/odin/highlights.scm:160]: makes sense then. Consider this resolved

**Review by lucario387 — APPROVED** [review]

**Pull request merged**